Formula/k/khal.rb

```diff
@@ -93,6 +93,9 @@ class Khal < Formula
 
   def install
     virtualenv_install_with_resources
+
+    generate_completions_from_executable(bin/"khal", shells: [:fish, :zsh], shell_parameter_format: :click)
+    generate_completions_from_executable(bin/"ikhal", shells: [:fish, :zsh], shell_parameter_format: :click)
```

Comment:
Will update to loop.

RuboCop issue needs fix on `brew` side to deal with nil handling (and probably group by executable)